sisyphus/localengine.py

```diff
@@ -71,7 +71,7 @@ class LocalEngine(threading.Thread, EngineBase):
     CPU and GPU are always checked, all other requirements only if given during initialisation.
     """
 
-    def __init__(self, cpus=1, gpus=0, **kwargs):
+    def __init__(self, cpus=1, gpus=0, available_gpus="", **kwargs):
```

Comment:
When a worker starts, it apparently (re-)creates its own version of `Localengine`. This leads to a problem with autodetecting the available GPUs from CUDA_VISIBLE_DEVICES 
(because in the worker this is now set to the specific devices that the worker should run on.)

Until anyone has some idea I add this as a new parameter.